tests/translated_table_model_tester.ts

```diff
@@ -11,6 +11,39 @@ function getTestTable() {
   return matrix;
 }
 
+function getLongTestTable() {
+  const matrix = new ArrayTableModel();
+  matrix.push([0]);
+  matrix.push([1]);
+  matrix.push([2]);
+  matrix.push([3]);
+  matrix.push([4]);
+  matrix.push([5]);
+  matrix.push([6]);
+  matrix.push([7]);
+  return matrix;
+}
+
+function shuffleRows(table: TranslatedTableModel) {
+  table.moveRow(7, 0);
+  table.moveRow(6, 1);
+  table.moveRow(5, 2);
+  table.moveRow(3, 4);
+}
+
+function areTableCellsEqual(table: TableModel, expectedTable: any[][]) {
+  for(let i = 0; i < table.rowCount; ++i) {
```

Comment:
When extending your unit testing framework, do it in a manner consistent with the framework. Refer to:

https://github.com/alsatian-test/alsatian/wiki/extending-expect

For how to do so.

Consider that this is a function that only ever returns ```true```, which makes its return value meaningless.